src/test/java/com/google/cloud/tools/gradle/appengine/core/CheckCloudSdkTaskTest.java

```diff
@@ -88,7 +91,9 @@ public void testCheckCloudSdkAction_versionMismatch() {
   }
 
   @Test
-  public void testCheckCloudSdkAction_sdkInstallationException() {
+  public void testCheckCloudSdkAction_sdkInstallationException()
```

Comment:
I think this is my mistake, but this should probably be named something more specific since it's only checking for CloudSdkNotFoundException.

Reply:
It looks like we're testing a little too deep here (we probably only need to check that we're calling the checks we want), their behavior is tested by the appengine-plugins-core tests.